service/src/main/resources/config.template.properties

```diff
@@ -20,4 +20,9 @@
   activedirectory.managerPassword : ENC(<encrypted password>) or password
   activedirectory.managerDn:
   activedirectory.rootDn :
-  activedirectory.userSearchFilter:
\ No newline at end of file
+  activedirectory.userSearchFilter:
+
+#ER lookup configurations
+
+  event-repository.enabled:<true/false>
+  event-repository.url :<ER URL for lookup>
```

Comment:
Remove "for lookup" from the comment. just ER base Url,, and specific Event-Repository rest entrypoint name/path should be in the Java code. If ER is started with context path,, then user add context path to the ER base url.

Reply:
Updated the comment to give generalized information.

Reply:
ok